pkg/handler/collector/oci/oci_test.go

```diff
@@ -0,0 +1,186 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package oci
+
+import (
+	"context"
+	"errors"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/guacsec/guac/internal/testing/dochelper"
+	"github.com/guacsec/guac/internal/testing/testdata"
+	"github.com/guacsec/guac/pkg/handler/processor"
+)
+
+// TODO (parth): Another way to testing the polling functionality? Currently
+// context cancel fails as context is used by regclient and causes a timeout
+func Test_ociCollector_RetrieveArtifacts(t *testing.T) {
+	ctx := context.Background()
+	type fields struct {
+		repo     string
+		tag      string
+		poll     bool
+		interval time.Duration
+	}
+
+	tests := []struct {
+		name       string
+		fields     fields
+		wantErr    bool
+		errMessage error
+		want       []*processor.Document
+	}{{
+		name: "multi-platform sbom",
+		fields: fields{
+			repo:     "ppatel1989/go-multi-test",
```

Comment:
could be a nice good first issue for someone to move this to a ghcr.io container under guacsec.

Reply:
Seems like we cannot have public images under the guacsec org. I tried that but dont have permission to change it. You or @mihaimaruseac might have to change that.

Reply:
done!

Reply:
moved image to guacsec org